pkg/docker/events.go

```diff
@@ -54,6 +56,10 @@ func NewEventMonitor(portTracker *tracker.PortTracker) (*EventMonitor, error) {
 // MonitorPorts scans Docker's event stream API
 // for container start/stop events.
 func (e *EventMonitor) MonitorPorts(ctx context.Context) {
+	if err := e.initializeRunningContainers(ctx); err != nil {
+		log.Errorf("initializing exsisting container port mappings failed: %v", err)
```

Comment:
```suggestion
		log.Errorf("initializing existing container port mappings failed: %v", err)
```
Or a better rephrase:
```suggestion
		log.Errorf("failed to initialize existing container port mappings: %v", err)
```